specification/communication/data-plane/Microsoft.CommunicationServicesSms/stable/2021-03-07/communicationservicessms.json

```diff
@@ -0,0 +1,217 @@
+    "contact": {
+      "email": "acsdevexdisc@microsoft.com"
+    },
+    "version": "2021-03-07"
+  },
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/sms": {
+      "post": {
+        "tags": [
+          "Sms"
+        ],
+        "summary": "Sends a SMS message from a phone number that belongs to the authenticated account.",
+        "operationId": "Sms_Send",
+        "x-ms-examples": {
+          "SuccessfulSend": {
+            "$ref": "./examples/send.json"
+          }
+        },
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "body",
+            "name": "SendMessageRequest",
+            "description": "Represents the body of the send message request.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/SendMessageRequest"
+            }
+          },
+          {
+            "name": "api-version",
+            "in": "query",
+            "description": "Version of API to invoke.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted",
+            "schema": {
+              "$ref": "#/definitions/SmsSendResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "SmsRecipient": {
+      "description": "Recipient Details for sending SMS messages.",
+      "required": [
+        "to"
+      ],
+      "type": "object",
+      "properties": {
+        "to": {
+          "description": "The recipient's phone number in E.164 format.",
+          "type": "string"
+        },
+        "repeatability-request-id": {
+          "description": "If specified, the client directs that the request is repeatable; that is, the client can make the request multiple times with the same Repeatability-Request-ID and get back an appropriate response without the server executing the request multiple times. The value of the Repeatability-Request-ID is an opaque string representing a client-generated, 36-character hexadecimal case-insensitive encoding of a UUID (GUID), identifier for the request.",
+          "type": "string"
+        },
+        "repeatability-first-sent": {
+          "description": "MUST be sent by clients to specify that a request is repeatable. Repeatability-First-Sent is used to specify the date and time at which the request was first created.eg- Tue, 26 Mar 2019 16:06:51 GMT",
+          "type": "string"
```

Comment:
"format": "date-time"

Reply:
The timestamp should be in RFC3339 format: `yyyy-MM-ddTHH:mm:ssZ`

Reply:
![image](https://user-images.githubusercontent.com/9246598/106819158-42ca1200-662e-11eb-9b0a-01af70d4f7ad.png)

![image](https://user-images.githubusercontent.com/9246598/106819228-570e0f00-662e-11eb-975d-52141a97df71.png)

is given in 
![image](https://user-images.githubusercontent.com/9246598/106819257-60977700-662e-11eb-9ad4-c48fe46e9305.png)

Reply:
Oh I see. We normally use RFC3339 for Azure APIs, but I guess here makes sense to do RFC1123 maybe

Reply:
Thanks Reza, keeping as is for now.